src/main/java/no/nav/fo/veilarbregistrering/oppgave/Oppgave.java

```diff
@@ -1,9 +1,94 @@
 package no.nav.fo.veilarbregistrering.oppgave;
 
-public interface Oppgave {
+import no.nav.fo.veilarbregistrering.bruker.AktorId;
+import no.nav.fo.veilarbregistrering.orgenhet.Enhetsnr;
 
-    long getId();
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
 
-    String getTildeltEnhetsnr();
+public class Oppgave {
 
+    private static final Map<OppgaveType, String> beskrivelser = new HashMap<>(2);
+    static {
+        beskrivelser.put(
+                OppgaveType.OPPHOLDSTILLATELSE,
+                "Brukeren får ikke registrert seg som arbeidssøker pga. manglende oppholdstillatelse i Arena, " +
+                        "og har selv opprettet denne oppgaven. " +
+                        "Ring bruker og følg midlertidig rutine på navet om løsning for registreringen av arbeids- og oppholdstillatelse."
+        );
+        beskrivelser.put(
+                OppgaveType.UTVANDRET,
+                "Brukeren får ikke registrert seg som arbeidssøker fordi bruker står som utvandret i Arena, " +
+                        "og har selv opprettet denne oppgaven. " +
+                        "Ring bruker og følg vanlig rutine for slike tilfeller."
+
+        );
+    }
+
+    private final AktorId aktorId;
+    private final Enhetsnr enhetsnr;
+    private final OppgaveType oppgaveType;
+    private final LocalDate fristFerdigstilleles;
+    private final LocalDate aktivDato;
+
+    private Oppgave(AktorId aktorId, Enhetsnr enhetsnr, OppgaveType oppgaveType, LocalDate fristFerdigstilleles, LocalDate aktivDato) {
+        this.aktorId = aktorId;
+        this.enhetsnr = enhetsnr;
+        this.oppgaveType = oppgaveType;
+        this.fristFerdigstilleles = fristFerdigstilleles;
+        this.aktivDato = aktivDato;
+    }
+
+    public static Oppgave opprettOppgave(
+            AktorId aktorId,
+            Enhetsnr enhetsnr,
+            OppgaveType oppgaveType,
+            LocalDate dagensDato) {
+
+        return new Oppgave(aktorId, enhetsnr, oppgaveType, fristFerdigstilleles(dagensDato), dagensDato);
```

Comment:
Jeg lar objektet selv sørge for å opprette en gyldig versjon av seg selv, med minst mulig input.
Vi trenger ikke å teste fristFerdigstillelse i isolasjon. Det gjør vi gjennom dette objektet.